htdocs/expedition/class/expedition.class.php

```diff
@@ -238,7 +245,7 @@ function create($user)
 		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
 		$sql.= ")";
 
-		dol_syslog(get_class($this)."::create", LOG_DEBUG);
+		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
```

Comment:
i did not know.
If changes are made taht impact developpement methods and tolls (like 
sys_log function), it should be correctly annouced to all and 
documentated on wiki to avoid such problems, and waste time for nothing 
(you and me).

Do you want me to made the correction, or will you do it ?

Le 02/04/2015 09:03, Marcos García a écrit :
>
> In htdocs/expedition/class/expedition.class.php 
> <https://github.com/Dolibarr/dolibarr/pull/2513#discussion_r27636066>:
>
> > @@ -238,7 +245,7 @@ function create($user)
> >  		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
> >  		$sql.= ")";
> >
> > -		dol_syslog(get_class($this)."::create", LOG_DEBUG);
> > +		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
>
> The thing that is being logged is the query. That line is used to 
> recognize which function called the query. Like a separator.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Dolibarr/dolibarr/pull/2513/files#r27636066>.
>